bower.json

```diff
@@ -14,7 +14,7 @@
   "dependencies": {
     "o-colors": "^3.3.0",
     "o-fonts": "^2.1.1",
-    "o-typography": "^3.1.0",
+    "o-typography": "^4.3.0",
```

Comment:
That would make this a major release, could you instead update it to `>=3.1.0 <5` instead, which will resolve the issue and allow this to be a patch release instead.